api/envoy/extensions/filters/network/postgres_proxy/v3alpha/postgres_proxy.proto

```diff
@@ -29,4 +29,14 @@ message PostgresProxy {
   // are parsed. Parsing is required to produce Postgres proxy filter
   // metadata. Defaults to true.
   google.protobuf.BoolValue enable_sql_parsing = 2;
+
+  // Controls whether to terminate SSL session initiated by a client.
+  // If the value is false, the Postgres proxy filter will not try to
+  // terminate SSL session, but will pass all the packets to the upstream server.
+  // If the value is true, the Postgres proxy filter will try to terminate SSL
+  // session. In order to do that, the filter chain must use :ref:`starttls transport socket
+  // <envoy_api_msg_extensions.transport_sockets.starttls.v3.StartTlsConfig>`.
+  // If the filter does not manage to terminate the SSL session, it will close the connection from the client.
+  // Defaults to false.
```

Comment:
Just make this a regular `bool` and no need to state the default as it's implied.

Reply:
Modified to use `bool`. Thanks. 
Quick question: previously when I run `proto_format.sh fix` it propagated changes forward to v4alpha. But not this time. Is it because api version here is v3alpha? Maybe I should change it to v3?

Reply:
The reason it's not doing this is that there is no type dependency that would cause a breaking change in v4 and we don't automatically bump such protos. We should, but that's a tooling issue you don't need to deal with, I'll comment in https://github.com/envoyproxy/envoy/issues/12841.